src/docs/product/data-management-settings/server-side-sampling/getting-started.mdx

```diff
@@ -24,6 +24,7 @@ Server-side sampling is currently only available for the following SDKs:
 - JavaScript: 7.6.0 or later
 - Apple: 7.23.0 or later
 - Android: 6.2.1 or later
+- React Native: 4.3.0 or later
```

Comment:
@imatwawana How about we have this list in an include? I feel we'll eventually go out of sync here

Reply:
Yeah, good call, I can set that up. Coincidentally, we only got the ability to use an include on the product side of our docs a couple weeks ago.